src/drivers/driver_zephyr.c

```diff
@@ -1131,6 +1131,14 @@ static int wpa_drv_zep_get_capa(void *priv,
 	capa->flags |= WPA_DRIVER_FLAGS_SAE;
 	capa->rrm_flags |= WPA_DRIVER_FLAGS_SUPPORT_RRM;
 
+	capa->enc |= WPA_DRIVER_CAPA_ENC_WEP40 |
+			WPA_DRIVER_CAPA_ENC_WEP104 |
+			WPA_DRIVER_CAPA_ENC_TKIP |
+			WPA_DRIVER_CAPA_ENC_CCMP |
+			WPA_DRIVER_CAPA_ENC_CCMP |
+			WPA_DRIVER_CAPA_ENC_CCMP_256 |
+			WPA_DRIVER_CAPA_ENC_GCMP_256;
+
```

Comment:
Sorry for doing this in multiple passes..but I was just thinking....shouldn't this be gathered from the device driver (i.e. have a op defined which will return the capabilities which a particular device supports)?

Reply:
Absolutely, I did think of this and glanced but couldn't see any event from RPU, so, skipped it. But we can re-look or implement one, but may in another PR?

Reply:
Yeah we presently don't have a way to get this information from the RPU but I was thinking for now lets atleast move this from driver_zephyr.c to zephyr_wpa_supp_if.c to make the wpa_supplicant code device agnostic. We can then have a TODO in zephyr_wpa_supp_if.c to get this from the RPU at a later stage.